exchanges/alert/alert_test.go

```diff
@@ -108,3 +109,37 @@ func BenchmarkWait(b *testing.B) {
 		n.Wait(nil)
 	}
 }
+
+// not needed in binary
```

Comment:
rm or update to something like 
```suggestion
// getSize checks the buffer size for testing purposes
```